pkg/system/system.go

```diff
@@ -167,8 +170,19 @@ func (sc *Controller) registerRouter() {
 	sc.router.HandleFunc(endpointDaemons, sc.describeDaemons()).Methods(http.MethodGet)
 	sc.router.HandleFunc(endpointDaemonsUpgrade, sc.upgradeDaemons()).Methods(http.MethodPut)
 	sc.router.HandleFunc(endpointDaemonRecords, sc.getDaemonRecords()).Methods(http.MethodGet)
+	sc.router.HandleFunc(endpointPrefetch, sc.getPrefetchFiles()).Methods(http.MethodGet)
+}
+func (sc *Controller) getPrefetchFiles() func(w http.ResponseWriter, r *http.Request) {
+	return func(w http.ResponseWriter, r *http.Request) {
+		body, _ := io.ReadAll(r.Body)
+		err := json.Unmarshal(body, &globalvar.Msg)
```

Comment:
Deserializing the body to a global variable is not safe. We need to reconsider how to manage the prefetch files list